Assets/ConveyorUpdate.cs

```diff
@@ -0,0 +1,22 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class ConveyorUpdate : MonoBehaviour {
+	
+	public bool conveyorMovedL = false;
```

Comment:
well it makes sense both ways, this way the variable says whether it has already moved the doll or not and if a conveyor belt has already moved a doll left it won't move the doll left again.

Your name would also make sense but in that case I would also have to change the parity of all of the bools, if you think it would be clearer that way I can do that though.